{
  "NServiceBus.MessageId": "2349d553-9f03-4770-9136-a81900ffb5c7",
  "NServiceBus.MessageIntent": "Send",
  "NServiceBus.ReplyToAddress": "ClientUI",
  "NServiceBus.CorrelationId": "2349d553-9f03-4770-9136-a81900ffb5c7",
  "NServiceBus.OriginatingMachine": "RINGMASTER-PC",
  "NServiceBus.OriginatingEndpoint": "ClientUI",
  "$.diagnostics.originating.hostid": "0cab1f2e91644ae2b0eab7216ca7a47a",
  "NServiceBus.ContentType": "text\/xml",
  "NServiceBus.EnclosedMessageTypes": "Messages.PlaceOrder, Messages, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
  "NServiceBus.ConversationId": "c520ec69-2494-4b89-bbf0-a81900ffb5d1",
  "NServiceBus.Version": "6.4.2",
  "NServiceBus.TimeSent": "2017-10-27 15:31:00:747219 Z",
  "NServiceBus.ExceptionInfo.ExceptionType": "System.InvalidOperationException",
  "NServiceBus.Retries.Timestamp": "2017-10-27 15:33:03:275486 Z",
  "NServiceBus.ExceptionInfo.HelpLink": null,
  "NServiceBus.ExceptionInfo.Message": "No handlers could be found for message type: Messages.PlaceOrder",
  "NServiceBus.ExceptionInfo.Source": "NServiceBus.Core",
  "NServiceBus.ExceptionInfo.StackTrace": "System.InvalidOperationException: No handlers could be found for message type: Messages.PlaceOrder\u000d\u000a   at NServiceBus.LoadHandlersConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\LoadHandlersConnector.cs:line 31\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.DeserializeLogicalMessagesConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\DeserializeLogicalMessagesConnector.cs:line 33\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.ReceivePerformanceDiagnosticsBehavior.<Invoke>d__2.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Performance\\Statistics\\ReceivePerformanceDiagnosticsBehavior.cs:line 40\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.ProcessingStatisticsBehavior.<Invoke>d__0.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Performance\\Statistics\\ProcessingStatisticsBehavior.cs:line 27\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.TransportReceiveToPhysicalMessageProcessingConnector.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\Incoming\\TransportReceiveToPhysicalMessageProcessingConnector.cs:line 37\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at NServiceBus.MainPipelineExecutor.<Invoke>d__1.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Pipeline\\MainPipelineExecutor.cs:line 32\u000d\u000a--- End of stack trace from previous location where exception was thrown ---\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\u000d\u000a   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)\u000d\u000a   at NServiceBus.LearningTransportMessagePump.<ProcessFile>d__9.MoveNext() in C:\\BuildAgent\\work\\a93f853f0c1b9532\\src\\NServiceBus.Core\\Transports\\Learning\\LearningTransportMessagePump.cs:line 277",
  "NServiceBus.TimeOfFailure": "2017-10-27 15:33:55:478829 Z",
  "NServiceBus.FailedQ": "ClientUI",
  "NServiceBus.ProcessingMachine": "RINGMASTER-PC",
  "NServiceBus.ProcessingEndpoint": "ClientUI",
  "$.diagnostics.hostid": "0cab1f2e91644ae2b0eab7216ca7a47a",
  "$.diagnostics.hostdisplayname": "RINGMASTER-PC"
}